src/Components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -159,7 +167,7 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
   const handleSubmit = async (e: any) => {
     const validForm = validateForm();
 
-    if (validForm) {
+    if (!validForm) {
```

Comment:
@patelaryan7751 can you please double check if it should be `!validForm`

Reply:
Yes, before it was not working as expected. So I changed it to make it work as expected.